src/status_im2/contexts/chat/lightbox/zoomable_image/view.cljs

```diff
@@ -204,76 +204,72 @@
+                                                 :pinch-y-start (anim/use-val c/init-offset)
+                                                 :pinch-y       (anim/use-val c/init-offset)
+                                                 :pinch-x-max   (anim/use-val js/Infinity)
+                                                 :pinch-y-max   (anim/use-val js/Infinity)
+                                                 :rotate        (anim/use-val c/init-rotation)
+                                                 :rotate-scale  (anim/use-val c/min-scale)}
+        props                                   {:pan-x-enabled? (reagent/atom false)
+                                                 :pan-y-enabled? (reagent/atom false)
+                                                 :focal-x        (reagent/atom nil)
+                                                 :focal-y        (reagent/atom nil)}
+        rescale                                 (fn [value exit?]
+                                                  (utils/rescale-image value
+                                                                       exit?
+                                                                       dimensions
+                                                                       animations
+                                                                       props))]
+    (rn/use-effect (fn []
+                     (js/setTimeout #(reset! set-full-height? true) 500)))
+    (when platform/ios?
+      (utils/handle-orientation-change curr-orientation focused? dimensions animations props)
+      (utils/handle-exit-lightbox-signal exit-lightbox-signal
+                                         index
+                                         (anim/get-val (:scale animations))
+                                         rescale
+                                         set-full-height?))
+    (utils/handle-zoom-out-signal zoom-out-signal index (anim/get-val (:scale animations)) rescale)
+    (let [tap (tap-gesture #(on-tap portrait?))
+          double-tap
+          (double-tap-gesture dimensions animations rescale transparent? #(on-tap portrait?))
+          pinch
+          (pinch-gesture dimensions animations props rescale transparent? #(on-tap portrait?))
+          pan-x (pan-x-gesture dimensions animations props rescale)
+          pan-y (pan-y-gesture dimensions animations props rescale)
+          composed-gestures (gesture/exclusive
+                             (gesture/simultaneous pinch pan-x pan-y)
+                             (gesture/exclusive double-tap tap))]
+      [gesture/gesture-detector {:gesture composed-gestures}
+       [reanimated/view
+        {:style (style/container dimensions
+                                 animations
+                                 @set-full-height?
+                                 (= curr-orientation orientation/portrait))}
+        [reanimated/fast-image
+         {:source    {:uri (:image content)}
```

Comment:
The component was previously two functions. It now has become one function, isn't this going to cause unnecessary re-renders?

And we now have a second `let` binding in the same function, which makes it redundant